### DISCUSSION TITLE
File size and ignore_patterns

### DISCUSSION THREAD

**Discussion by Joouul** [phpro/grumphp#1094] [Q&A] [answered]
Hi,

I need to exclude big files but I want to commit all files in "images" folder even if size is more than 1M. There are multiple "images" folders in many dynamic folders like "/users/me/2022_01_01/images/" "/users/me/2022_06_24/images/"

grumphp:
  tasks:
    file_size:
      max_size: 1M
      ignore_patterns:
       - "/users/me/**/images/*"

I've tried many options without success ...

Anoyone have a solution ?
THX

**Comment by veewee** [comment, marked as answer]
Hello,

This configuration uses the symfony Finder path() method:
* https://symfony.com/doc/current/components/finder.html
* https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/Finder/Finder.php#method_path

It doesn't accept globs - but it does accept regular expressions.
Also: you might want to drop the '/' in front - since that indicates an absolute path.

**Reply by Joouul** [reply]
Thank for indication :) It's work :)
